packages/types/src/scripts/generate/tx.ts

```diff
@@ -45,6 +49,20 @@ function tsDoc (documentation: Vec<Text>): string {
   return docs === '' ? '' : indent(6)(`/**\n${indent(6)(indent(1)(`* ${docs.toString().trim().replace(/^\w/, c => c.toUpperCase()).replace(/\.$/, '')}`))}\n${strarg}${indent(7)('*/')}\n`);
 }
 
+// Generate namespaces file
+/** @internal */
+function generateNamespaces (moduleName: string, methods: FunctionMetadataV10[]): void {
```

Comment:
generateTxTags

Reply:
Also, ideally this function should build and return the contents of the file instead of modifying an external variable